packages/app-mobile/components/Icon.tsx

```diff
@@ -34,8 +34,9 @@ const Icon: React.FC<Props> = props => {
 	const importantForAccessibility = accessibilityHidden ? 'no-hide-descendants' : 'yes';
 
 	const sharedProps = {
-		importantForAccessibility,
-		'aria-hidden': accessibilityHidden,
+		importantForAccessibility, // Android
+		accessibilityElementsHidden: accessibilityHidden, // iOS
+		'aria-hidden': accessibilityHidden, // Web
```

Comment:
This fixes an issue where the folder icon to the left of the "Notebooks" heading was accessibility focusable on iOS (with no label or description). On Android and Web, the same icon is hidden for accessibility.